#888 - Implemented & Recategorized Sensing Blocks

rpowley opened the pull request:
Implemented and re-categorized sensing blocks: Ask, Pointer X&Y, Random X&Y, Center X&Y, Stage Height/Width, Key Pressed.

dethe commented on js/event.js:
```diff
@@ -191,11 +191,11 @@
+                //    evt.invalid = true;
+                //    evt.invalidReason = 'Not the left mouse button';
+                //    return evt;
+                //}
```
We can go ahead and delete these lines, not just comment them out.

eddieantonio commented:
Apart from the weird whitespace changes, I think everything's looking good. For the most part, I really like the new organization of these blocks, though I agree that the random point, random X, and random Y should go in Canvas, since they are relative to the canvas. Also, the fact that random X and random Y have nothing to do with Point (neither taking in a Point, nor returning a Point).

rpowley commented:
I'm not sure why the white spaces changed like that because it looks normal locally? 

I deleted the commented bit and moved randomX and randomY to canvas

dethe commented:
The whitespace is probably from editor settings. Not a huge problem, but worth looking into the config when you get the chance.
